### PR TITLE
Add DA properties and pages

### PR DESCRIPTION
**Pull request by torztomasz**
This pull request adds DA properties to config and a page to website.

### CODE REVIEW

**Comment by lucadonnoh** [comment]
wdyt about changing "5/7 DAC Members" to just "5/7 DAC"

**Comment by torztomasz** [comment]
> wdyt about changing "5/7 DAC Members" to just "5/7 DAC"

I would keep it as it is cause its clearer for me this way, but will talk about this with guys

File: packages/config/src/layer2s/templates/opStack.ts [hunk 1]
```diff
@@ -158,6 +157,19 @@ export function opStack(templateVars: OpStackConfig): Layer2 {
           : templateVars.finality,
     },
     chainConfig: templateVars.chainConfig,
+    dataAvailability:
+      templateVars.daProvider !== undefined
+        ? makeDataAvailabilityConfig({
+            type: 'Off chain',
+            layers: ['Celestia', 'Ethereum (calldata)'],
```
Comment (antooni) [inline packages/config/src/layer2s/templates/opStack.ts:164]: In OP stack projects Celestia will always be the provider? Is it safe to use it here as default value?

File: packages/frontend/src/components/header/PageHeader.tsx [hunk 2]
```diff
@@ -2,10 +2,14 @@ import React from 'react'
 
 import { HorizontalSeparator } from '../HorizontalSeparator'
 
-export function RiskHeader() {
+interface PageHeaderProps {
+  children: string
+}
+
+export function PageHeader({ children }: PageHeaderProps) {
```
Comment (antooni) [inline packages/frontend/src/components/header/PageHeader.tsx:9]: SimplePageHeader?

**Review by maciekzygmunt — CHANGES_REQUESTED** [review]
Please add DA to sidebar menu
image

**Review by maciekzygmunt — APPROVED** [review]

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
risk rosette tooltips are broken

File: packages/config/src/common/dataAvailability.ts [hunk 3]
```diff
@@ -1,160 +1,260 @@
...
+  }
+}
+
+function DAC_SENTIMENT(DAC?: {
+  membersCount: number
+  requiredSignatures: number
+}) {
+  if (
+    DAC &&
+    (DAC.membersCount < 6 || DAC.requiredSignatures / DAC.membersCount < 1 / 3)
```
Comment (lucadonnoh) [inline packages/config/src/common/dataAvailability.ts:250]: this formula is wrong, it should use the size - threshold + 1

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Comment by lucadonnoh** [comment]
https://www.notion.so/l2beat/DA-risks-d55883d11bd843c781f2242bd32d8467

image

**Review by lucadonnoh — APPROVED** [review]

File: packages/backend/discovery/nova/ethereum/diffHistory.md [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+
+# Diff at Fri, 08 Mar 2024 11:12:54 GMT:
+
+- author: torztomasz (<tomasz.torz@l2beat.com>)
+- comparing to: main@f09f798ebd2ae57f4c76e08114d608edf0a51c7b block: 19132347
+- current block number: 19390070
+
+## Description
+
+Provide description of changes. This section will be preserved.
```
Comment (lucadonnoh) [inline packages/backend/discovery/nova/ethereum/diffHistory.md:11]: provide the description that the handler has been changed and not that the actual values have been changed, do the same for others

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Review by luke7211 — DISMISSED** [review]
Fraxtal has a wrong DA

File: packages/config/src/layer2s/templates/opStack.ts (hunk 1 above)
Comment (luke7211) [reply packages/config/src/layer2s/templates/opStack.ts:175]: No

File: packages/backend/discovery/nova/ethereum/diffHistory.md (hunk 4 above)
Comment (torztomasz) [reply packages/backend/discovery/nova/ethereum/diffHistory.md:11]: It can not be added due to some bug, ask @mateuszradomski about it.

**Comment by luke7211** [comment]
* Remove "Or None" from DA Layer
* Fix Fraxtal

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
- Celestia should be yellow
- @vincfurc will check Metis DA challenge mechanism, because rn Metis DA risk is Red but on this page both columns are yellow and it's not consistent

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**